Document template permissions checks gotcha (#17)
dabapps/django-zen-queries: README.md

If you absolutely definitely can't avoid running a query in a part of your codebase that's being executed under a `queries_disabled` block, there is another context manager called `queries_dangerously_enabled` which allows you to temporarily re-enable database queries.

### Permissions gotcha

> If you are using permissions in your templates (via the `{{ perms }}` template variable), this can be a source of queries at template-render time. Fortunately, Django's permission checks are [cached by the `ModelBackend`](https://docs.djangoproject.com/en/2.2/topics/auth/default/#permission-caching), so we can pre-populate the cache by calling `request.user.get_all_permissions()` in the view, before rendering the template.

Reviewer: There's some strange changes in tense and context

```suggestion
Accessing permissions in your templates (via the `{{ perms }}` template variable) can be a source of queries at template-render time. Fortunately, Django's permission checks are [cached by the `ModelBackend`](https://docs.djangoproject.com/en/2.2/topics/auth/default/#permission-caching), which can be pre-populated by calling `request.user.get_all_permissions()` in the view, before rendering the template.
```
